matchms/importing/load_adducts.py

```diff
@@ -1,11 +1,13 @@
+import json
 import os
 from functools import lru_cache
 from typing import Dict
+import pandas as pd
 import yaml
 
 
 @lru_cache(maxsize=4)
-def load_adducts(filename: str = None) -> Dict:
+def load_adducts_dict(filename: str = None) -> Dict:
     """Load dictionary of known adducts. Makes sure that file loading is cached.
```

Comment:
Please document what happens when filename=None. 
```suggestion
    """Load dictionary of known adducts. Makes sure that file loading is cached.
    If no argument is given or None is given then the `data/known_adducts.yaml` file from the `matchms` package is loaded.
```